trame_rca/protocol.py

```diff
@@ -68,6 +68,8 @@ def push_content(self, area_name, metadata, content):
     def on_interaction(self, area_name, origin, event):
         adapter = self._area_adapters.get(area_name, None)
         if adapter:
-            adapter.on_interaction(origin, event)
+            # FIXME
+            print("on_interaction", event)
+            # adapter.on_interaction(origin, event)
```

Comment:
@ChristosT this is where you need to connect the server side... Probably need to uncomment the code and fix the async part to understand the events you get.